### PR TITLE
Fix encoding

### LINKED ISSUE #7: Escaped unicode
q4qVmWUU@2x

\u2605 (★) is sneaking through

### PR DESCRIPTION
**Pull request by cottongin**
fixes #7

### CODE REVIEW

File: src/irc/__init__.py [hunk 1]
```diff
@@ -130,6 +131,7 @@ def _get(data, key, format_found=None, default=None):
     if key in data:
         value = data[key]
         if value:
+            value = _sanitize(value)
```
Comment (valcanobacon) [inline src/irc/__init__.py]: Can you move this to the outside again?

**Review by valcanobacon — CHANGES_REQUESTED** [review]
Thanks this looks gooood :+1: Can you move the _santizie call again and ill merge. A unittest for _sanitize would be amaaaazing. Thanks!

File: src/irc/__init__.py (hunk 1 above)
Comment (cottongin) [reply src/irc/__init__.py]: ...sigh no idea how that got back in there, #[21b5574](https://github.com/valcanobacon/BoostBots/pull/10/commits/21b5574adebf168ddd72f073c71b62fa4fe61d3a) fixes

Comment (cottongin) [reply src/irc/__init__.py]: As to unittest, is that blocking merging this?

**Pull request merged**